src/app/utils/reverseItemFilterMap.js

```diff
@@ -0,0 +1,7 @@
+const getLabelForValue = (map, value) => {
```

Comment:
You can delete this file. The `getLabelForValue` function is not being used but there is another one defined and used in `itemFilterUtils.js` that should be kept.

Reply:
done